fli-iam/shanoir-ng
# shanoir-issue#2224: rename solr column + new column with da.importDate

How to test:
Simply make a new import

In solr, 'creation date' was renamed to 'series date'
Also in solr: two new column named 'import date' (date of creation of the dataset_acquisition) and 'imported by' (with the username of the user who did the import)
In dataset acquisition detail, those two data are also visible:
![image](https://github.com/fli-iam/shanoir-ng/assets/117283961/f152b774-f1d1-4ce7-bd20-4df345ceaea2)

And in solr:
![image](https://github.com/fli-iam/shanoir-ng/assets/117283961/a02dbe81-726f-4632-8dd5-71cc4cd26ae4)


In order to make sure solr is up to date, please run follow the doc here: https://github.com/fli-iam/shanoir-ng/wiki/Add-a-new-Solr-search-facet#qualif-instances
